packages/data-stores/src/reader/hooks/index.ts

```diff
@@ -0,0 +1,14 @@
+import { useSelector } from 'react-redux';
```

Comment:
`react-redux` is not in the dependency tree of the `@automattic/data-stores` . Hence we get the [error](https://teamcity.a8c.com/buildConfiguration/calypso_WPComPlugins_HappyBlocks/9677490?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildProblemsSection=true&expandBuildChangesSection=true&showLog=9677490_403_348&logFilter=debug&logView=flowAware).